### PR TITLE
Add studentInfo intent

### PR DESCRIPTION
**Pull request by gabrielGavriluta**

### CODE REVIEW

**Review by raiuRobert — APPROVED** [review]

File: intents/user/intentInitStudentInfoEN.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+    "Hi! I am [StudX]. I am a student in group [Gx], series [Sx], semigroup [SmX]. The current semester is [SemX] and the week is [WTx].",
+    "Hello! My name is [StudX]. I am a student in group [Gx], series [Sx], semigroup [SmX]. The semester is [SemX], and the week is [WTx].",
+    "Hi! My name is [StudX], student in group [Gx], series [Sx], semigroup [SmX]. The current semester is [SemX], and the study week is [WTx].",
+    "Hi! I am [StudX], I am a student in the group [Gx] [SmX], series [Sx]. The current semester is [SemX] and the parity is [WTx].",
+    "Hi! My name is [StudX]. I am a student in the group [Gx] [Sx] [SmX]. It is the semester [SemX], and the week is [WTx].",
+    "Hello! My name is [StudX]. I am a student in group [Gx], series [Sx], semigroup [SmX]. The semester is [SemX], and the week is [WTx].",
+    "Hello! My name is [StudX]. I am a student in the group [Gx] [Sx] [SmX]. The semester is [SemX], and the week is [WTx].",
+    "Hello! I am [StudX], a student in the group [Gx] [Sx] [SmX]. The semester is [SemX], and the week is [WTx]."
```
Comment (ancadavid) [inline intents/user/intentInitStudentInfoEN.json:9]: You don't annontate the specified entities correctly.  You can look at the other intentions defined by me (for example getStartTimeEvent). 
For more details you can check the documentation: https://rasa.com/docs/rasa/nlu-training-data/

File: intents/user/parse_student_info.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import os
+
+with open('intentInitStudentInfoRO.json') as intentSRLeaderRO:
+    intentRO = json.load(intentSRLeaderRO)
+with open('intentInitStudentInfoEN.json') as intentSRLeaderEN:
+    intentEN = json.load(intentSRLeaderEN)
+with open('../../../resources/nrSR.json') as nrSRModel:
+    nrSR = json.load(nrSRModel)
+f = open("generatedIntentSRLeaderRO.yml", "w")
+fEN = open("generatedIntentSRLeaderEN.yml", "w")
```
Comment (ancadavid) [inline intents/user/parse_student_info.py:11]: You generate intentions for initStudentInfo, but still the files generated by you are for SRLeader intention. Change the names.

File: intents/user/intentInitStudentInfoRO.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
+    "Bună! Eu sunt [StudX]. Sunt student in grupa [Gx], seria [Sx], semigrupa [SmX]. Semestrul curent este [SemX], iar săptămâna este [WTx].",
+    "Salut! Numele meu este [StudX]. Sunt student in grupa [Gx], seria [Sx], semigrupa [SmX]. Semestrul este [SemX], iar săptămâna este [WTx].",
+    "Bună! Mă numesc [StudX], student in grupa [Gx], seria [Sx], semigrupa [SmX]. Semestrul curent este [SemX], iar săptămâna de studii este [WTx].",
+    "Bună! Eu sunt [StudX], sunt student in grupa [Gx][SmX], seria [Sx]. Semestrul curent este [SemX], iar paritatea este [WTx].",
+    "Bună! Numele meu este [StudX]. Sunt student in grupa [Gx][Sx][SmX]. Este semestrul [SemX], iar săptămâna este [WTx].",
+    "Salut! Numele meu este [StudX]. Sunt student in grupa [Gx], seria [Sx], semigrupa [SmX]. Semestrul este [SemX], iar săptămâna este [WTx].",
+    "Salut! Mă numesc [StudX]. Sunt student in grupa [Gx][Sx][SmX]. Semestrul este [SemX], iar săptămâna este [WTx].",
+    "Salut! Eu sunt [StudX], student in grupa [Gx][Sx][SmX]. Semestrul este [SemX], iar săptămâna este [WTx]."
```
Comment (ancadavid) [inline intents/user/intentInitStudentInfoRO.json:9]: Same problem

**Review by ancadavid — CHANGES_REQUESTED** [review]

File: intents/user/intentInitStudentInfoEN.json [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
+    "Hi! My name is [StudX](student), student in group [Gx](groupNr), series [Sx](SrNr), semigroup [SmX](semigroup). The current semester is [SemX](semester), and the study week is [WTx](weekType).",
+    "Hi! I am [StudX](student), I am a student in the group [Gx](groupNr)[SmX](semigroup), series [Sx](SrNr). The current semester is [SemX](semester) and the parity is [WTx](weekType).",
+    "Hi! My name is [StudX](student). I am a student in the group [Gx](groupNr) [Sx](SrNr) [SmX](semigroup). It is the semester [SemX](semester), and the week is [WTx](weekType).",
+    "Hello! My name is [StudX](student). I am a student in group [Gx](groupNr), series [Sx](SrNr), semigroup [SmX](semigroup). The semester is [SemX](semester), and the week is [WTx](weekType).",
+    "Hello! My name is [StudX](student). I am a student in the group [Gx](groupNr)[Sx](SrNr) [SmX](semigroup). The semester is [SemX](semester), and the week is [WTx](weekType).",
+    "Hello! I am [StudX](student), a student in the group [Gx](groupNr)[Sx](SrNr)[SmX](semigroup). The semester is [SemX](semester), and the week is [WTx](weekType)."
```
Comment (ancadavid) [inline intents/user/intentInitStudentInfoEN.json:9]: srNr

File: intents/user/intentInitStudentInfoRO.json [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
+    "Bună! Mă numesc [StudX](student), student in grupa [Gx](groupNr), seria [Sx](SrNr), semigrupa [SmX](semigroup). Semestrul curent este [SemX](semester), iar săptămâna de studii este [WTx](weekType).",
+    "Bună! Eu sunt [StudX](student), sunt student in grupa [Gx](groupNr)[SmX](semigroup), seria [Sx](SrNr). Semestrul curent este [SemX](semester), iar paritatea este [WTx](weekType).",
+    "Bună! Numele meu este [StudX](student). Sunt student in grupa [Gx](groupNr)[Sx](SrNr)[SmX](semigroup). Este semestrul [SemX](semester), iar săptămâna este [WTx](weekType).",
+    "Salut! Numele meu este [StudX](student). Sunt student in grupa [Gx](groupNr), seria [Sx](SrNr), semigrupa [SmX](semigroup). Semestrul este [SemX](semester), iar săptămâna este [WTx](weekType).",
+    "Salut! Mă numesc [StudX](student). Sunt student in grupa [Gx](groupNr)[Sx](SrNr)[SmX](semigroup). Semestrul este [SemX](semester), iar săptămâna este [WTx](weekType).",
+    "Salut! Eu sunt [StudX](student), student in grupa [Gx](groupNr)[Sx](SrNr)[SmX](semigroup). Semestrul este [SemX](semester), iar săptămâna este [WTx](weekType)."
```
Comment (ancadavid) [inline intents/user/intentInitStudentInfoRO.json:9]: srNr

**Review by ancadavid — CHANGES_REQUESTED** [review]

**Pull request merged**